man/join_ald_scenario.Rd

```diff
@@ -12,29 +12,11 @@ join_ald_scenario(
   add_green_technologies = FALSE
 )
 }
-\arguments{
-\item{data}{A data frame like the output of
-\code{r2dii.match::prioritize}.}
-
-\item{ald}{An asset level data frame like \link[r2dii.data:ald_demo]{r2dii.data::ald_demo}.}
-
-\item{scenario}{A scenario data frame like \link[r2dii.data:scenario_demo_2020]{r2dii.data::scenario_demo_2020}.}
-
-\item{region_isos}{A data frame like \link[r2dii.data:region_isos]{r2dii.data::region_isos} (default).}
-
-\item{add_green_technologies}{Logical vector of length 1. \code{FALSE} defaults to
-outputting only technologies that are present in both \code{data} and \code{ald}. Set
-to \code{FALSE} to add rows of all possible green technologies (with 0
-production).}
-}
-\value{
-Returns a fully joined data frame, linking portfolio, ald and
-scenario.
-}
 \description{
-\code{join_ald_scenario()} is a simple wrapper of several calls to
-\verb{dplyr::join_*()}, forming the master dataset to be used in later steps of
-the analysis.
+\ifelse{html}{\href{https://lifecycle.r-lib.org/articles/stages.html#deprecated}{\figure{lifecycle-deprecated.svg}{options: alt='[Deprecated]'}}}{\strong{[Deprecated]}}
+
+This function was deprecated in favour of \code{\link[=join_abcd_scenario]{join_abcd_scenario()}}. See \href{https://2degreesinvesting.github.io/posts/2022-03-02-ald-becomes-abcd/}{here}
```

Comment:
nit: do we use British English (e.g. favour) by default?

Reply:
Good question... I think I always defaulted to it since it is also Canadian spelling lol